CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Move isseraddress computation into its own function.
```

Comment:
Small typo: "Move `isseraddress` computation into its own function"

Reply:
Also, the CI currently fails due to reported vulnerabilities

Reply:
nice catch, fixed it!